File: sdkwork-backend-sdk-go/types/vad_config.go
package types


type VadConfig struct {
	SpeechThreshold float64 `json:"speechThreshold"`
	MinSpeechDurationMs int `json:"minSpeechDurationMs"`
	MinSilenceDurationMs int `json:"minSilenceDurationMs"`
	MaxSilenceDurationMs int `json:"maxSilenceDurationMs"`
	SilenceTimeout int `json:"silenceTimeout"`
	SpeechTimeout int `json:"speechTimeout"`
	WindowSize int `json:"windowSize"`
	StepSize int `json:"stepSize"`
}

File: sdkwork-backend-sdk-csharp/Models/VadConfig.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class VadConfig
    {
        public double? SpeechThreshold { get; set; }
        public int? MinSpeechDurationMs { get; set; }
        public int? MinSilenceDurationMs { get; set; }
        public int? MaxSilenceDurationMs { get; set; }
        public int? SilenceTimeout { get; set; }
        public int? SpeechTimeout { get; set; }
        public int? WindowSize { get; set; }
        public int? StepSize { get; set; }
    }
}
